library/tests/src/com/google/maps/android/kml/KmlContainerParserTest.java

```diff
@@ -0,0 +1,76 @@
+package com.google.maps.android.kml;
+
+import android.test.ActivityTestCase;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserFactory;
+
+import java.io.InputStream;
+
+import com.google.maps.android.test.R;
+
+/**
+ * Created by lavenderch on 1/27/15.
+ */
+public class KmlContainerParserTest extends ActivityTestCase {
+
+    public XmlPullParser createParser(int res) throws Exception {
+        InputStream stream = getInstrumentation().getContext().getResources().openRawResource(res);
+        XmlPullParserFactory factory = XmlPullParserFactory.newInstance();
+        factory.setNamespaceAware(true);
+        XmlPullParser parser = factory.newPullParser();
+        parser.setInput(stream, null);
+        parser.next();
+        return parser;
+    }
+
+    public void testCDataEntity() throws Exception {
+        KmlContainerParser parser = new KmlContainerParser(createParser(R.raw.cdata));
+        KmlContainer kmlContainer = new KmlContainer();
+        parser.assignContainerProperties(kmlContainer);
+        assertEquals(kmlContainer.getKmlProperty("description"), "TELEPORT");
```

Comment:
assert* should have the expected value first, then the observed value. This is used in error messages (e.g. "Test X failed, expected "TELEPORT" actual "SOMETHINGELSE"")